pve-5.2-2-diff-backup-addon

```diff
@@ -420,24 +420,36 @@ revert)
         ;;
 
 apply)
-        if pve_manager_patch | patch_apply /usr/share/pve-manager/ --dry-run --batch
-        then
-                pve_manager_patch | patch_apply /usr/share/pve-manager/
-                echo "PATCHED: /usr/share/pve-manager/"
-        fi
-        if pve_perl_patch | patch_apply /usr/share/perl5/PVE/ --dry-run --batch
-        then
-                pve_perl_patch | patch_apply /usr/share/perl5/PVE/
-                echo "PATCHED: /usr/share/perl5/PVE/"
+        PKG_OK=$(dpkg-query -W --showformat='${Status}\n' patch 2>/dev/null |grep "install ok installed")
```

Comment:
I would make it simpler:

```bash
if ! which patch >/dev/null; then
  echo "patch: missing, please install with 'apt-get install patch'"
  exit 1
fi

case $1 in
revert)
```

Reply:
👍  I only don't like "exit-ing" :-)

Reply:
Well, there's no point of running `patch` unless you have it :)